packages/tools/devtools/devtools-browser-extension/package.json

```diff
@@ -103,21 +113,29 @@
 		"eslint-plugin-chai-expect": "~3.0.0",
 		"good-fences": "^1.1.1",
 		"html-webpack-plugin": "^5.5.0",
+		"jest": "^26.6.3",
+		"jest-dev-server": "^4.3.0",
```

Comment:
Is this dep needed? I don't see it referenced.

Reply:
jest-dev-server is used when jest-puppeteer-config start a server for testing.